en/flight_controller/kakuteh7mini.md

```diff
@@ -0,0 +1,141 @@
+# Holybro Kakute H7 mini
+
+:::warning
+PX4 does not manufacture this (or any) autopilot.
+Contact the [manufacturer](https://shop.holybro.com/) for hardware support or compliance issues.
+:::
+
+The [Holybro Kakute H7 mini](http://www.holybro.com/product/kakute-h7-mini/) flight controller is full of features including integrated Bluetooth, HD camera plug, dual plug-and-play 4in1 ESC ports, 9V VTX ON/OFF Pit Switch, barometer, OSD, 6x UARTs, 128MB Flash for Logging (not supported with PX4 yet), 5V and 9V BEC, and bigger soldering pad with easy layout and much more.
```

Comment:
" (not supported with PX4 yet)" - is logging still not supported?

```suggestion
The [Holybro Kakute H7 mini](http://www.holybro.com/product/kakute-h7-mini/) flight controller is full of features including integrated Bluetooth, HD camera plug, dual plug-and-play 4in1 ESC ports, 9V VTX ON/OFF Pit Switch, barometer, OSD, 6x UARTs, 128MB Flash for logging (not supported with PX4 yet), 5V and 9V BEC, and bigger soldering pad with easy layout and much more.
```

Reply:
It's not planned to be supported for now.